CMakeLists.txt

```diff
@@ -37,6 +42,7 @@ option(UMF_BUILD_BENCHMARKS "Build UMF benchmarks" OFF)
 option(UMF_BUILD_BENCHMARKS_MT "Build UMF multithreaded benchmarks" OFF)
 option(UMF_BUILD_EXAMPLES "Build UMF examples" ON)
 option(UMF_BUILD_GPU_EXAMPLES "Build UMF GPU examples" OFF)
+option(UMF_LINK_HWLOC_STATICALLY "Link UMF with HWLOC library statically" OFF)
```

Comment:
please add static linking to the windows build matrix in CI workflow

Reply:
Done.